pypit/armlsd.py

```diff
@@ -171,6 +176,7 @@ def ARMLSD(fitsdict, reuseMaster=False, reloadMaster=True):
                     tilts, satmask, outpar = artrace.multislit_tilt(slf, slf._msarc[det-1], det)
                     slf.SetFrame(slf._tilts, tilts, det)
                     slf.SetFrame(slf._satmask, satmask, det)
+                    # This outpar is only the last slit!!  JXP doesn't think it matters for now
```

Comment:
Should we use a `msgs.work()` or `msgs.bug()` here. Alternatively, we could include a `TODO` here to remember to come back to it at some point.

Reply:
made it a msgs.bug